packages/client/utils/reactjiPickerUtils.ts

```diff
@@ -0,0 +1,28 @@
+const DEFAULT = {
+  heart: 9,
+  tada: 8,
+  smile: 7,
+  rocket: 6,
+  fire: 5,
+  white_check_mark: 4,
+  confused: 3,
+  cry: 2,
+  x: 1
+}
+
+const getRecent = () => {
+  const localStorageRecent = localStorage.getItem('recent')
+  return localStorageRecent === null ? DEFAULT : JSON.parse(localStorageRecent)
+}
+
+const handleRecent = (emoji) => {
```

Comment:
Please add a type to the argument since our goal is to have noImplicitAny rule turned on.

Reply:
Naming can be improved, like `updateUsage`